mappings/net/minecraft/packet/play/UseBedS2C.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_234 net/minecraft/packet/play/UseBedS2C
```

Comment:
StartSleepingS2C. This packet would still work if a mod added a non-bed, wouldn't it?

Reply:
yes i thonk so